patroni/postgresql/bootstrap.py

```diff
@@ -106,11 +111,11 @@ def option_is_allowed(name):
             error_handler('{0} options must be list or dict'.format(tool))
         return user_options
 
-    def _initdb(self, config):
+    def _initdb(self, config: Any) -> bool:
```

Comment:
```suggestion
    def _initdb(self, config: Union[Any, Dict[str, str], List[Union[str, Dict[str, Any]]]]) -> bool:
```

Just so we use the same annotation of `options` argument in `process_user_options`.

Reply:
I tried it, but as soon as it becomes Union we have to explicitly assert types in respective methods :(